.goreleaser.prerelease.yml

```diff
@@ -12,15 +12,17 @@ builds:
       - linux
     goarch:
       - amd64
+      - arm64
     ldflags:
-      - '-w -s -X github.com/chanzuckerberg/fogg/util.GitSha={{.Commit}} -X github.com/chanzuckerberg/fogg/util.Version={{.Version}} -X github.com/chanzuckerberg/fogg/util.Dirty=false -X github.com/chanzuckerberg/fogg/util.Release=true'
+      - '-w -s -X main.Date={{.CommitDate}} -X github.com/chanzuckerberg/fogg/util.GitSha={{.Commit}} -X github.com/chanzuckerberg/fogg/util.Version={{.Version}} -X github.com/chanzuckerberg/fogg/util.Dirty=false -X github.com/chanzuckerberg/fogg/util.Release=true'
```

Comment:
` -X main.Date={{.CommitDate}}`

Why this change?

Reply:
It was a recommended change on the Goreleaser docs if you want builds to be reproducible from a given commit. Relying on the commit date/time rather than the system time means that we get the _exact_ same metadata on the binary when we build it locally vs on CI.

Reply:
This may not get us a perfectly reproducible build, but it's one less thing that'll be different from builds of the same commit.